Lib/calendar.py

```diff
@@ -286,8 +317,8 @@ def yeardayscalendar(self, year, width=3):
         Day numbers outside this month are zero.
         """
         months = [
```

Comment:
monthdayscalendar one line (Part 1)
```suggestion
        months = [self.monthdayscalendar(year, m) for m in Month]
```